pkg/pgtune/wal.go

```diff
@@ -14,8 +14,7 @@ const (
 
 	walBuffersThreshold = 2 * parse.Gigabyte
 	walBuffersDefault   = 16 * parse.Megabyte
-	minWALBytes         = 4 * parse.Gigabyte
-	maxWALBytes         = 8 * parse.Gigabyte
+	defaultMaxWALBytes  = 8 * parse.Gigabyte
```

Comment:
I don't think we should be limiting the `max_wal_size`; for a very active instance we need to tune the checkpoints to be very far apart, say 30 minutes and an `max_wal_size=8GB` may too small for that.

Example:

- High througput instance, 10k's transactions per second
- 100MB WAL / second

To tune this instance to have 30 minutes checkpoints requires at least 100MB/s*60s*30 ≈ 176GB

I vote for having an unbound maximum, as with this feature in, users can limit the `max_wal_size` by passing on the max wal disk size already.

Reply:
Okay, we can do that. Do you have a preference on what the default should be if the user doesn't provide a wal disk size?

Reply:
Tough question, PostgreSQL defaults to `1GB`:

https://www.postgresql.org/docs/current/runtime-config-wal.html#GUC-MAX-WAL-SIZE

We may actually want to avoid setting the parameter if we think `1GB` is sensible.

TimescaleDB is certainly going to take advantage of larger `max_wal_size` settings on very active instances, however, there will be users running this on a constrained environment, so increasing the default may cause issues there.